README.md

```diff
@@ -32,6 +37,51 @@ Meeting Agenda templates
 - Meeting - Sprint Retrospective, Review, and Planning
 - Meeting - Sprint Open Topic Session 
 
+## Running the project locally
+To start the project locally type
+
+
+The following list of things are required to use the software:
+
+- npm
+  ```sh
+  npm install npm@latest -g
+
+- Clone the repo
+   ```sh
+   git clone https://github.com/chingu-voyages/v46-tier3-team-33/tree/readme4
+   ```
+- Install NPM packages
```

Comment:
the npm install must be done in both the frontend and the backend directories not the root directory - can you expand this instruction so it reflects that please?